lib/core/src/Cardano/Wallet.hs

```diff
@@ -908,7 +907,6 @@ restoreBlocks ctx wid blocks nodeTip = db & \DBLayer{..} -> mapExceptT atomicall
 
     liftIO $ do
         progress <- walletSyncProgress @ctx @s ctx (NE.last cps)
-        traceWith tr $ MsgWalletMetadata meta
```

Comment:
We could perhaps tweak the `MsgSyncProgress` to include a wallet name and possibly the active delegation status. We don't need the time interpreter for those, and it's still quite valuable to log those information when debugging logs.